datetimepickeredittext/src/main/java/io/blackbox_vision/datetimepickeredittext/internal/utils/DateUtils.kt

```diff
@@ -40,4 +40,12 @@ object DateUtils {
 
         return date
     }
+
+    private fun currentDT(): String {
```

Comment:
`currentDT` seems like difficult to understand if I don't really see the code of the function. Can we change the name to `getCurrentDateTime`, it's more longer, but it's more expressive.

Reply:
Yes, we can change this. As, its private function that's why it can't reflect any issue to users it's only for developer to understand the function functionality.

Reply:
Yes! I know, but my intention is also focus on DX, which means that if someone wants to contribute, needs to understand the code. 

Would you mind to change this?

Reply:
modified and pushed the code